scripts/init

```diff
@@ -201,19 +152,29 @@ if [ -n "$POSTANALYTIC_ADDRESS" ] && [ -n "$POSTANALYTIC_ONLY" ]; then
 fi
 
 if [ -z "$WALLARM_API_TOKEN" ]; then
-  if [ ! -f "/etc/wallarm/private.key" ]; then
+  if [ ! -f "/opt/wallarm/etc/wallarm/private.key" ]; then
     echo "ERROR: no WALLARM_API_TOKEN and no private key in /etc/wallarm/private.key" >&2
     exit 1
   fi
 fi
 
+# Propagate env vars to supervisor
+set -a && source /opt/wallarm/env.list && set +a
```

Comment:
Could you please double check this?
I am getting such an error when running it:
```
root@4ac664631da1:/# set -a && source /opt/wallarm/env.list && set +a
bash: /opt/wallarm/usr/local/share/tarantool/?/init.lua: No such file or directory
bash: /opt/wallarm/usr/local/share/wallarm-tarantool/modules/?.lua: No such file or directory
bash: /opt/wallarm/usr/local/share/wallarm-tarantool/modules/?/init.lua: No such file or directory
```
which makes Tarantool quit.

Using `export $(sed -e 's/=\(.*\)/="\1"/g' /opt/wallarm/env.list | grep -v "#" | xargs)` fixes the issue.

At least this was an issue when running with `env.list` from the `4.8.6` version. Another fix is to quote the `LUA_PATH` value.

Reply:
@nedvna take a look please

Reply:
Hello @pbaclawski,

We patch this in the 4.10 image, achieved this way: https://github.com/wallarm/docker-wallarm-node/pull/53/files#diff-fa455f75ee8dccc747b5dccba98d3aca6eb6751da07161e057d6abe60d9af05fR24-R26

The way 4.8 stable is shipped is quite different, so this fix was not required there
Not sure how you got that error on 4.8 but you can use the fix from 4.10

Please note this PR is still WIP though

Reply:
Thank you, @nedvna!

I haven't seen this fix, hence the issue, with it, everything seems to work as expected. Thank you once again. :)